doc/nrf/ug_multi_image.rst

```diff
@@ -86,6 +86,9 @@ You can enable existing child images in the |NCS| by enabling the respective mod
 To turn an application that you have implemented into a child image that can be included in a parent image, you must update the build scripts to enable the child image and add the required configuration options.
 You should also know how image-specific variables are disambiguated and what targets of the child images are available.
 
+Enabling an existing image through Kconfig can not be done through menuconfig, but must instead be done by modifying Kconfig fragments, such as :file:`prj.conf`.
+This is due to image configuration being interdependent.
+
```

Comment:
afaik this has never been an issue before.

Is this only happening with TF-M and in combination with nRF Security ?
At least from the referenced issue, I see that TF-M is build with nRF Security, and thus it could be related to the TF-M build.

Note, TF-M is not included as a regular child image build, but as an external project directly in Zephyr.

Reply:
placed a comment in the referred issue on why this happens for the TF-M build.

In general users should be able to enable or disable a child image, which will cause CMake to re-run.
If that doesn't work, it's an issue and we must investigate why it fails.